app/main.ipynb

```diff
@@ -100,18 +92,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "arr = np.arange(10, dtype=np.float64)\n",
+    "arr[[0, 2, 9]]"
```

Comment:
The array `arr` should have a length between 5 and 10, but `np.arange(10)` creates an array of length 10. Consider using a smaller range or slicing the array to meet the requirement.